### ISSUE TITLE
Envelope rates returned for items thicker than 1in

### ISSUE THREAD

**Issue by madeincosmos** [Automattic/woocommerce-services#1218]
#### Affected tickets

781375-zen

#### What I expected

To ship a mug over USPS (4x6x6in)

#### What happened instead

I got a Large Envelope rate returned for this item. According to USPS specs, Large Envelope can't be more than 3/4 inches thick:

https://pe.usps.com/businessmail101?ViewName=Flats

I tried changing **Packing Method** in the plugin settings, but this didn't help much - happens for both items packed individually and standard boxes.

#### Steps to reproduce the issue

1. Add a USPS shipping method and enable all services,
2. Create a product (3lbs, 4x6x6in) and add it to cart,
3. Observe the Large Envelope rate is returned:

![Screenshot](http://cld.wthms.co/S3YJYw+)
Screenshot: http://cld.wthms.co/S3YJYw

Reproduced on Pressable, with WC 3.2.3 & Services 1.8.2

**Comment by jeffstieler** [comment]
@madeincosmos I was able to reproduce this with both WCS and the USPS shipping method extension. It seems that there's a bug on the USPS API side that's returning First-Class Mail Large Envelope for all requests.

![cart store name here - http___local wordpress dev_cart_](https://user-images.githubusercontent.com/63922/32617142-3339fb1a-c542-11e7-8741-908e1d8f9357.png)

I've reached out to our contacts at USPS to report the issue and cc'd you. Thanks for the detailed bug report! 👍
